src/main/kotlin/blackjack/domain/player/Player.kt

```diff
@@ -1,51 +1,71 @@
 package blackjack.domain.player
 
 import blackjack.domain.card.Card
-import blackjack.domain.card.Card.AceCard
-import blackjack.domain.game.TakeMorePlayerStrategy
+import blackjack.domain.card.ReceivedCards
+import blackjack.domain.game.strategy.TakeMorePlayerStrategy
 
 open class Player(
     private val _name: String,
-    val takeMorePlayerStrategy: TakeMorePlayerStrategy,
-    val receivedCards: MutableSet<Card> = mutableSetOf()
+    private val _receivedCards: ReceivedCards = ReceivedCards(mutableSetOf()),
+    private val _gambleSummary: GambleSummary = GambleSummary()
 ) {
-
-    var isWinner: Boolean = false
+    constructor(name: String, cards: List<Card>) : this(_name = name) {
+        cards.map {
+            _receivedCards.addCard(it)
+        }
+    }
 
     val score: Int
-        get() = calculateScore()
+        get() = _receivedCards.calculateScore().score
 
     val name: String
         get() = _name
 
-    fun calculateScore(): Int {
-        var score = receivedCards.sumOf { it.number }
+    val receivedCards: ReceivedCards
+        get() = _receivedCards
 
-        if (score > BLACKJACK_SCORE) {
-            val aceCount = receivedCards
-                .count { it is AceCard }
+    val gambleSummary: GambleSummary
+        get() = _gambleSummary
 
-            score = score - (ACE_NUMBER_TO_ELEVEN * aceCount) + (ACE_NUMBER_TO_ONE * aceCount)
-        }
+    fun canMoreGame(): Boolean {
+        return _receivedCards.calculateScore().score < BLACKJACK_SCORE
+    }
 
-        return score
+    fun isBust(): Boolean {
+        return _receivedCards.calculateScore().score > BLACKJACK_SCORE
     }
 
-    fun canMoreGame(): Boolean {
-        return calculateScore() < BLACKJACK_SCORE
+    fun isBlackJack(): Boolean {
+        return _receivedCards.count() == CARD_SIZE_FOR_BLACKJACK && _receivedCards.calculateScore().score == BLACKJACK_SCORE
```

Comment:
```suggestion
        return _receivedCards.isBlackJack()
```
속성을 가진 객체에게 위임해 봅시다. 플레이어 입장에서는 블랙잭 여부만 중요하지 블랙잭이 두장의 카드를 가져야 하는지는 중요하지 않다고 생각해요.